docusaurus/docs/Android/01-basics/02-choosing-the-correct-sdk.mdx

```diff
@@ -0,0 +1,246 @@
+  which parts of the UI you want to keep original, and which you want to customize.
+
+### Extensive Layout Changes
+
+What do we mean by that? Put simply, it signifies the ability to re-order elements in a Composable/ View or even replace parts of a Composable/ View with your custom UI.
+
+In the traditional View based technology, achieving this is difficult, incurs a significant maintenance cost and can lead to inefficient UI due to multiple measurement passes. The combination of slots in Compose and its firm rule to measure layouts only once allows us to give you the ability to significantly alter the UI.
+
+Let's see that in action:
+
+![MessageListHeader Slots](../assets/choosing_the_correct_sdk_message_list_header_slots.png)
+
+Notice how the image shows the header broken down into 3 distinct pieces?
+All of these are slots which allow you to pass in your own content.
+
+We are used to seeing small changes in XML, such as changing the background color, setting a different drawable inside an `ImageView` or so on, but Compose enables much more radical changes.
+
+In just dozen simple lines, you can leverage Compose to re-order and customize the content of this header like so:
+
+![Custom MessageListHeader component](../assets/compose_custom_message_list_header_component.png)
+
+If you're curious about the code powering these changes, you can read more about it [here](http://localhost:3000/chat/docs/sdk/android/compose/message-components/message-list-header/#customization).
+
+`MessageListHeader` is not the only Composable we offer that uses slots, in fact, most of our major components such as `MessageComposer`, `MessageList`, various menus, dialogs and others make extensive use of slots.
+This level of customization is miles above anything we can offer you with UI Components.
+
+### Results Conclusion
+
+If you want a stock chat experience that works out of the box, either SDK is a good option. However, if you want a highly customizable chat, our Compose SDK is unparalleled in this regard.
+
+Don't feel apprehensive about trying it out if your developers are not familiar with Jetpack Compose and/or if your app is written using XML and Views.
+Jetpack Compose is easy to learn and is inter-operable with View based technology. The time saved by not having to write boilerplate code such as `ViewHolder`s will most likely offset the time spent on learning Jetpack Compose. On top of that, Compose code is often easier to maintain and produces fewer bugs.
+
+## Implementation Cost
+
+Implementation cost will be reflected upon from our experience implementing our chat tutorial applications.
+If you want to check out how to build those, take a look at the [UI Components](https://getstream.io/tutorials/android-chat/) and [Compose](https://getstream.io/chat/compose/tutorial/) tutorials.
+
+| SDK | Time Cost | Development Cost |
+|:---:|:-------------:|:-----: |
+| UI Components | Good | 4 files, 180 line insertions, 12 line deletions |
```

Comment:
A quantitative time difference here was not representative of the real world cost. The dev rel team did a great job writing them and they are very easy to follow so they lessen the difference. 

A blind implementation would show the difference better, but would leave the users having to simply trust us.

I think being honest with a qualitative description + including a verifiable difference in code changes creates a much more realistic picture and allows the users to check for themselves.